angr/analyses/analysis.py

```diff
@@ -189,18 +171,16 @@ class Analysis:
     _progress_callback = None
     _show_progressbar = False
     _progressbar = None
-    _statusbar: Optional[StatusBar] = None
+    _task = None
 
     _PROGRESS_WIDGETS = [
-        progressbar.Percentage(),
-        " ",
-        progressbar.Bar(),
-        " ",
-        progressbar.Timer(),
-        " ",
-        progressbar.ETA(),
-        " ",
-        StatusBar(),
+        rich.progress.TaskProgressColumn(),
+        rich.progress.BarColumn(),
+        rich.progress.TextColumn("Elapsed Time:"),
+        rich.progress.TimeElapsedColumn(),
+        rich.progress.TextColumn("Time:"),
+        rich.progress.TimeRemainingColumn(),
+        rich.progress.TextColumn("{task.description}"),
```

Comment:
How about
```python
from rich import progress
```
```python
progress.TaskProgressColumn(),
progress.BarColumn(),
progress.TextColumn("Elapsed Time:"),
progress.TimeElapsedColumn(),
progress.TextColumn("Time:"),
progress.TimeRemainingColumn(),
progress.TextColumn("{task.description}"),
```

Reply:
I thought about doing this, but I didn't want anything to accidentally override the progress keyword as a local variable somewhere. It's probably okay to do (and looks a bit cleaner) but it's such a common name that I avoided it.

Reply:
👍Oh I see, thanks

Reply:
I looked through it again and it seemed fine to do, so I did it :)